apps/airdrop/components/claim/ClaimTokenScreen.tsx

```diff
@@ -0,0 +1,90 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { useAccount } from 'wagmi';
+
+import { AlreadyClaimedStep } from './components/AlreadyClaimedStep';
+import { DonationConfirmationStep } from './components/DonationConfirmationStep';
+import type { DonationPercentage } from './components/DonationSelectionStep';
+import { DonationSelectionStep } from './components/DonationSelectionStep';
+import { NotQualifiedStep } from './components/NotQualifiedStep';
+import { ShowClaimableTokensStep } from './components/ShowClaimableTokensStep';
+import { StartClaimStep } from './components/StartClaimStep';
+import { TokenClaimSuccessStep } from './components/TokenClaimSuccessStep';
+
+const DEV_TOKEN_AMOUNT = 2500;
```

Comment:
this will be replaced by some kind of lookup right?

Reply:
correct